pkg/ui/v1alpha3/nas.go

```diff
@@ -13,8 +13,8 @@ import (
 
 func (k *KatibUIHandler) FetchAllNASJobs(w http.ResponseWriter, r *http.Request) {
 	//enableCors(&w)
-	// Use "" to get experiments in all namespaces.
-	jobs, err := k.getExperimentList("", JobTypeNAS)
+	// Get NAS-related experiments in all available namespaces.
+	jobs, err := k.getExperimentList(availableNameSpaces, JobTypeNAS)
```

Comment:
What do you expect when you have multiple namespaces passed?

Reply:
Currently, in Katib Client: https://github.com/kubeflow/katib/blob/master/pkg/util/v1alpha3/katibclient/katib_client.go#L210, we extract only first namespace from the list, so it will not give you all available namespaces.

Reply:
@andreyvelich Yes, but value of namespace is fixed on "" (https://github.com/kubeflow/katib/blob/master/pkg/ui/v1alpha3/hp.go#L22) which leads to list experiments in cluster scope.

Reply:
@johnugeorge It seems only first namespace will be taken. It is a problem.